Doc/docker config (#1415)
kestra-io/docs: content/docs/02.installation/13.docker-compose.md

If you want to extend your Docker Compose file, modify container networking, or if you have any other issues using this Docker Compose file, check the [Troubleshooting Guide](/docs/faq/troubleshooting).
::

### Use a configuration file

If you want to use a configuration file instead of the `KESTRA_CONFIGURATION` environment variable to configure Kestra you can update the default `docker-compose.yml`.

> First, create a configuration file, for example named `application.yaml` and put inside the content of the `KESTRA_CONFIGURATION` environment variable defined in the `docker-compose.yml` file.
> 
> Then update `kestra` servce in the `docker-compose.yml` file to mount this file into the container and make Kestra using it via the `--config` option:

Reviewer: ```suggestion
First, create a configuration file containing the `KESTRA_CONFIGURATION` environment variable defined in the `docker-compose.yml` file. You can name it `application.yaml`.

Then, update `kestra` service in the `docker-compose.yml` file to mount this file into the container and make Kestra using it via the `--config` option:
```
